common/resource_manager.cc

```diff
@@ -520,4 +531,137 @@ bool ResourceManager::CheckAllowNavigation(const std::string& url) {
   return result = false;
 }
 
+bool ResourceManager::isEncrypted(const std::string& path) {
+  auto setting = application_data_->setting_info();
+  if (setting.get() == NULL)
+    return false;
+
+  if (setting->encryption_enabled()) {
+    std::string ext = utils::ExtName(path);
```

Comment:
ExtName should be fixed. 

if path was "file:///abc.html?test=c"

return value was "html?test=c"